ofrak_core/ofrak/core/magic.py

```diff
@@ -22,19 +22,54 @@ class Magic(ResourceAttributes):
     descriptor: str
 
 
+class _LibmagicDependency(ComponentExternalTool):
+    try:
+        import magic
+
+        magic_: ModuleType = magic
+    except ImportError:
+        magic_ = None
```

Comment:
Instead of duplicating this try-catch logic, why not have a global `MAGIC_INSTALLED` variable like we are already doing with [binwalk](https://github.com/redballoonsecurity/ofrak/blob/master/ofrak_core/ofrak/core/binwalk.py)?

Reply:
@EdwardLarson